messages/purge/messages.go

```diff
@@ -0,0 +1,14 @@
+package purge
+
+var (
+	Usage            = "purge"
+	ShortDescription = "Purge domains"
+	LongDescription  = "Purge domains"
+	FlagHelp         = "Displays more information about the purge command"
+	FlagWildcard     = ""
+	FlagCacheKeys    = ""
+	FlagUrls         = ""
+	FlagLayer        = "Fields: 'edge_caching, l2_caching'"
```

Comment:
```suggestion
	FlagLayer        = "Informs the layer the purge will be executed. Possible values: 'edge_caching' or 'l2_caching'"
```

Reply:
`L2 Caching` shouldn't be `Tiered Caching`?
(Not sure if the API still use `l2`, but the UI surely uses `Tiered` nowadays)

Reply:
@PabloDiehl documentation mentioned l2... not sure about the API